### PR TITLE
feat(eventing): snapshot and clone events

### PR DESCRIPTION
**Pull request by datacore-vvarakantham**
Added snapshot and clone events.

### CODE REVIEW

File: apis/events/protobuf/v1/event.proto [hunk 1]
```diff
@@ -210,3 +216,21 @@ message ReactorEventDetails {
   // Reactor state
   string state = 2;
 }
+
+// Snapshot event details
+message SnapshotEventDetails {
+  // Parent id of the snapshot
+  string parent_id = 1;
```
Comment (tiagolobocastro) [inline apis/events/protobuf/v1/event.proto]: what is the parent id?

Comment (datacore-vvarakantham) [reply apis/events/protobuf/v1/event.proto]: Parent_id is replica_id. And entity_id is volume_id. Should I rename the variables?

Comment (tiagolobocastro) [reply apis/events/protobuf/v1/event.proto]: Yes I think would be helpful here

Comment (datacore-vvarakantham) [reply apis/events/protobuf/v1/event.proto]: Changed the variable names to replica_id and volume_id.

**Review by tiagolobocastro — APPROVED** [review]

**Review by sinhaashish — APPROVED** [review]

**Pull request merged**